python/lsst/obs/ctio0m9/ingest.py

```diff
@@ -58,6 +58,35 @@ def translate_visit(self, md):
         """
         return mjdToVisit(md.get("DATE-OBS"))
 
+    def translate_imgType(self, md):
+        """Get the image type (e.g. bias, dark, flat etc) from the header
+        Translator function derived from a very small dataset from the observatory
+        i.e. may well need adding to when new string values are found
+        """
```

Comment:
Ideally docstrings are recommended to use the Numpydoc format, since there are plans to use sphinx and keep the code uniformly documented, but as I'm sure you've noticed this is not required and not always done.